ui/src/app/pages/profile/profile-page.tsx

```diff
@@ -153,17 +257,36 @@ export const ProfilePage = withUserProfile()(class extends React.Component<
     }
   }
 
+  validGivenName() {
```

Comment:
since this returns a boolean, could you name it something like `isGivenNameValid`?

Reply:
We actually do not need this method, since its explicitly validation the length of given name. Removing it